README.md

```diff
@@ -1,191 +1,58 @@
-identifiable.
-
-## Getting Started
-
-At the end of this tutorial, you will have initialized the Ably Chat client and sent your first message.
-
-First of all, start by creating a Swift project and installing the Chat SDK using the instructions described above. Next, replace the contents of your `main.swift` file with
-the following code. This simple script initializes the Chat client, creates a chat room and sends a message, printing it to the console when it is received over the websocket connection.
-
-```swift
-import Ably
-import AblyChat
-import Foundation
-
-// Create the Ably Realtime client using your API key and use that to instantiate the Ably Chat client.
-// You can re-use these clients for the duration of your application
-let realtimeOptions = ARTClientOptions()
-realtimeOptions.key = "<API_KEY>"
-realtimeOptions.clientId = "ably-chat"
-let realtime = ARTRealtime(options: realtimeOptions)
-let chatClient = DefaultChatClient(realtime: realtime, clientOptions: nil)
-
-// Subscribe to connection state changes
-let connectionStateSubscription = chatClient.connection.onStatusChange()
-Task {
-    for await stateChange in connectionStateSubscription {
-        print("Connection status changed: \(stateChange.current)")
-    }
-}
-
-// Get a chat room for the tutorial
-let room = try await chatClient.rooms.get(
-    roomID: "readme-getting-started")
-
-// Add a listener to observe changes to the chat rooms status
-let statusSubscription = room.onStatusChange()
-Task {
-    for await status in statusSubscription {
-        print("Room status changed: \(status.current)")
-    }
-}
-
-// Attach the chat room - this means we will begin to receive messages from the server
-try await room.attach()
-
-// Add a listener for new messages in the chat room
-let messagesSubscription = try await room.messages.subscribe()
-Task {
-    // Subscribe to messages
-    for await message in messagesSubscription {
-        print("Message received: \(message.text)")
-    }
-}
+## Supported Platforms
 
-// Send a message
-_ = try await room.messages.send(
-    params: .init(text: "Hello, World! This is my first message with Ably Chat!"))
+Ably aims to support a wide range of platforms. If you experience any compatibility issues, open an issue in the repository or contact [Ably support](https://ably.com/support).
 
-// Wait 5 seconds before closing the connection so we have plenty of time to receive the message we just sent
-// This disconnects the client from Ably servers
-try await Task.sleep(nanoseconds: 5 * NSEC_PER_SEC)
-await chatClient.rooms.release(roomID: "readme-getting-started")
-realtime.close()
-print("Connection closed")
-```
+This SDK supports the following platforms:
 
-Now run your script:
+| Platform | Support      |
+|----------|--------------|
+| iOS      | >= 14.0      |
+| macOS    | >= 11.0      |
+| tvOS     | >= 14.0      |
 
-```shell
-swift run
-```
+> [!NOTE]
+> Xcode v16.1 or later is required.
```

Comment:
I'd suggest sticking to referring to products the way that Apple does (e.g. https://developer.apple.com/documentation/xcode-release-notes/xcode-16_1-release-notes):

```suggestion
> Xcode 16.1 or later is required.
```

Reply:
See [b61b91e](https://github.com/ably/ably-chat-swift/pull/298/commits/b61b91e0f5904d34715d9dfb7ca7a68447f4b640)